rfcs/text/0000-Incident-Response-Plan.md

```diff
@@ -30,23 +30,169 @@ As DCP governance, I want be notified of a confirmed or suspected incident so th
+
+##### Phase 3: Containment, Eradication, and Recovery
+This phase is used to gather electronic and physical evidence to determine exactly what has happened, who might be responsible, and how this type of intrusion can be prevented in the future.  The DCP IRT will make recommendations to remediate affected resources and verify completion. As necessary, the IRT will take measures to limit the amount of damage that an intruder or authorized affected system can inflict upon the rest of the IT infrastructure. 
+
+##### Phase 4: Post-Incident Activity
+Follow-up occurs in this phase. DCP IRT completes a report of the incident, subsequent recovery process, and lessons learned. A final copy of the incident report is sent to the Oversight Committee and appropriate stakeholders. Additionally, an incident post-mortem may be conducted to evaluate the lessons learned during the event.
+
+#### Incident Response Procedures
+Anyone can declare an incident. Follow the procedure outlined below if you suspect or have confirmed that an incident has occurred.
+
+__Step 1__: Personnel identifying the suspected or confirmed incident (Incident Handler) will alert the DCP IRT by submitting a ticket to the to-be-created humancellatlas/dcp-security private GitHub repo. The ticket will include the following information to ensure appropriate triage:
+1. Assign the ticket to the DCP IRT Coordinator
+1. Date of discovery of the suspected or confirmed incident
+1. Brief description of suspected or confirmed incident
+1. If known, also include:
+   1. Date incident occurred
+   1. Date incident was discovered
+   1. Affected system(s) and application(s)
+   1. Number of users affected
+   1. Data types involved
+
+__Step 2__: Incident Handler will post an incident notification to the HumanCellAtlas/dcp-security Slack channel. The notification should not contain detailed information; instead, it should link to the incident’s tracking ticket. 
+
+Example: A suspected or confirmed incident has been identified and is being tracked in [insert_ticket_link_here].
+
+__Step 3__: DCP IRT will immediately review the submitted ticket and, with assistance from Incident Handlers, conduct a preliminary investigation of the event to determine if the event constitutes a security incident.
+* __Step 3.1__: If the DCP IRT determines that the event does not constitute a security incident or is a false positive, the DCP IRT Coordinator will document the determination and supporting evidence within the ticket and close the ticket. No further action is required.
+* __Step 3.2__: If the DCP IRT confirms that the event does constitute a security incident, the DCP IRT Coordinator will collaborate with the Incident Handler to save all evidence and complete an Incident Report (see Appendix B).
+
+__Step 4__: The DCP IRT Coordinator will update the incident’s ticket to include a link to the completed Incident Report and notify the following that the report has been completed:
```

Comment:
If it is something that requires a postmortem, the report-out could be valuable for the entire DCP.

Reply:
Sharing out to the rest of DCP after the incident response process is closed is perfectly acceptable.